manifests/a/Arm/ArmPerformanceLibraries/24.4/Arm.ArmPerformanceLibraries.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Arm.ArmPerformanceLibraries
+PackageVersion: "24.4"
+PackageLocale: en-US
+Publisher: Arm
+PackageName: Arm Performance Libraries
+License: BSD-3
```

```suggestion
Publisher: Arm
PublisherUrl: https://www.arm.com/
PackageName: Arm Performance Libraries
PackageUrl: https://developer.arm.com/Tools%20and%20Software/Arm%20Performance%20Libraries
License: BSD-3
```

另外我不清楚是从哪里得知此软件包的许可证是 BSD-3 的。